stories/LocationSelector.tsx

```diff
@@ -41,6 +41,7 @@ storiesOf('Organisms|LocationSelector', module)
 
         return (
             <LocationSelector
+                className="test-class"
```

Comment:
I guess it's forgotten to delete

Reply:
no, it is intentional. This is the error we had that inspired this PR. TS would throw when adding a className. So as a "test" I added it to storybook. I wonder if we shouldn't do it in every story, as this is not the first time we forgot to handle it. But that is for a separate discussion I think.